### DISCUSSION TITLE
Simpledifficulty Iron Canteens are not refillable from Rain Collectors.

### DISCUSSION THREAD

**Discussion by Ski-z** [ACGaming/UniversalTweaks#177] [Bugfix Suggestions]
[See the Rain Collector code here](https://github.com/Charles445/SimpleDifficulty/blob/51ffee8be7bc7dfd52b862ff208ed8428a5cbc00/src/main/java/com/charles445/simpledifficulty/block/BlockRainCollector.java)

[ItemCanteenIron](https://github.com/Charles445/SimpleDifficulty/blob/51ffee8be7bc7dfd52b862ff208ed8428a5cbc00/src/main/java/com/charles445/simpledifficulty/item/ItemIronCanteen.java) is not utilized in here for some reason, definitely a bug.

This is a paid commission! 👍

**Comment by ACGaming** [comment]
Implemented in f88102b63d9dc5b472ff1c11fb88064831c3f2c3.
